lib/smb2/packet/query_info_request.rb

```diff
@@ -35,6 +35,11 @@ class QueryInfoRequest < Smb2::Packet
     string :file_id, 128
 
     rest :buffer
+
+    # @return [Symbol] a key in {Smb2::COMMANDS}
+    def self.command
```

Comment:
Unnecessary if you move to [class-constant lookup](https://github.com/rapid7/smb2/pull/28/files#r28820364)